components/external_link/external_link.test.tsx

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {mount} from 'enzyme';
+import {screen} from '@testing-library/react';
+
+import {Provider} from 'react-redux';
+
+import {DeepPartial} from 'redux';
+
+import mockStore from 'tests/test_store';
+
+import {renderWithIntlAndStore} from 'tests/react_testing_utils';
+
+import {GlobalState} from 'types/store';
+
+import ExternalLink from '.';
+
+describe('components/external_link', () => {
+    const initialState: DeepPartial<GlobalState> = {
+        entities: {
+            general: {
+                config: {},
+                license: {
+                    Cloud: 'true',
+                },
+            },
+            users: {
+                currentUserId: 'currentUserId',
+            },
+        },
+    };
+
+    it('should match snapshot', () => {
+        const store = mockStore(initialState);
+        const wrapper = mount(
+            <Provider store={store}>
+                <ExternalLink
+                    href='https://mattermost.com'
+
+                >{'Click Me'}</ExternalLink>
+            </Provider>,
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it('should attach parameters if telemetry is enabled', () => {
+        const state = {
```

Comment:
Is this test still applicable?

Reply:
I updated the name of the test, it's applicable now!